src/Riverline/MultiPartParser/StreamedPart.php

```diff
@@ -36,18 +36,27 @@ class StreamedPart
      */
     private $parts = array();
 
+    /**
+     * The length of the EOL character.
+     *
+     * @var int
+     */
+    private $EOLCharacterLength = 2;
```

Comment:
You don't need to set a default value here because it will be override by the constructor

Reply:
Fixed.